### PR TITLE
New OutlinedField component

### PR DESCRIPTION
**Pull request by andreyvEze**
https://user-images.githubusercontent.com/22767467/120843440-c4976100-c576-11eb-8bda-21ff8dd68eb9.mov

### CODE REVIEW

File: src/components/themed/EdgeOutlinedField.js [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+      borderLeftColor: color
+    }
+  })
+  const animatedRightCornerStyle = useAnimatedStyle(() => {
+    const color = placeholderSize.value > 0 ? interpolateColor(colorMap.value, [0, 1, 2], [inactiveColor, activeColor, errorColor]) : inactiveColor
+    return {
+      borderTopColor: color,
+      borderRightColor: color
+    }
+  })
```
Comment (thehobbit85) [inline src/components/themed/EdgeOutlinedField.js:195]: Avoid this code duplication.
Instead, create a function like this as a const:
```
const cornerStyle = (borders: string[]) => () => {
    const color = placeholderSize.value > 0 ? interpolateColor(colorMap.value, [0, 1, 2], [inactiveColor, activeColor, errorColor]) : inactiveColor
    return borders.reduce((res, border) => ({ ...res, [border]: color}), {})
  }
```

And then you can use it to create the animatedStyles like this:


```
const animatedLeftCornerStyle = useAnimatedStyle(cornerStyle(['borderTopColor', 'borderLeftColor']))
```

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
see comments

**Comment by thehobbit85** [comment]
There is one more place that you need to replace the text component: 
https://github.com/EdgeApp/edge-react-gui/blob/37937adfc79a21fd07f2981b136aad2ede5a288e/src/components/themed/ManageTokensHeader.js#L44

It's the new manage token scene.

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comment

File: src/components/themed/EdgeOutlinedField.js [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+
+const CornerBorder = ({ theme, corner, cornerHeight, placeholderSize, colorMap }: CornerBorderProps) => {
+  const styles = getStyles(theme)
+  const { inactiveColor, activeColor, errorColor } = getSizeStyles(theme)
+
+  const animatedContainerStyles = useAnimatedStyle(() => {
+    const color =
+      placeholderSize.value > 0
+        ? interpolateColor(colorMap.value, [ANIMATION_STATES.INIT, ANIMATION_STATES.FOCUSED, ANIMATION_STATES.ERROR], [inactiveColor, activeColor, errorColor])
+        : inactiveColor
```
[the remark this answers is not in the document]
Comment (andreyvEze) [reply src/components/themed/EdgeOutlinedField.js]: ok, sure

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**